parallel-letter-frequency/example.rs

```diff
@@ -41,9 +40,9 @@ pub fn frequency(texts: &[&str], num_workers: usize) -> HashMap<char, usize> {
 fn count(lines: Vec<String>) -> HashMap<char, usize> {
     let mut results: HashMap<char, usize> = HashMap::new();
     for line in lines.iter() {
-        for c in line.as_slice().chars() {
+        for c in line.chars() {
             if c.is_alphabetic() {
-                match results.entry(c.to_lowercase()) {
+                match results.entry(c.to_ascii_lowercase()) {
```

Comment:
The test contains some non-7-bit-ASCII characters. I'm sure the tests pass with this but I'm not sure that's by accident or because it does the right thing.